data/timesketch.conf

```diff
@@ -259,6 +259,16 @@ YETI_API_KEY = ''
 # Labels to narrow down indicator selection
 YETI_INDICATOR_LABELS = ['domain']
 
+
```

Comment:
@tomchop @berggren from a design perspective consider decoupling contrib from the main code base

Reply:
For now, setting the expectation that any anlyzer in the contrib directory is out-of-scope for our e2e tests.